lib/widgets/compose_box.dart

```diff
@@ -735,9 +736,26 @@ class _SendButtonState extends State<_SendButton> {
 
     final store = PerAccountStoreWidget.of(context);
     final content = widget.contentController.textNormalized;
-    store.sendMessage(destination: widget.getDestination(), content: content);
 
     widget.contentController.clear();
+
+    try {
+      // TODO(#720) await send request, putting input(s) and send button into a
+      //   disabled "working on it" state (allowing input text to be selected
+      //   for copying).
+      await store.sendMessage(destination: widget.getDestination(), content: content);
```

Comment:
nit: first clause of the TODO sounds like a description of what the code is already doing